app/assets/stylesheets/projects/index.scss

```diff
@@ -138,27 +133,39 @@ $demo-btn-shadow: #3C93D5;
       height: 57px;
       line-height: 57px;
       overflow: hidden;
-      padding: 0 8px;
+      color: black;
+      padding: 0 10px;
+      margin-top: 5px;
     }
 
     .project-description {
       @include project-card-text;
       font-size: 16px;
       height: 48px;
-      line-height: 18px;
-      padding: 0 8px;
+      line-height: 24px;
+      padding: 0 10px;
```

Comment:
Properties should be ordered font-size, height, line-height, margin-bottom, padding